CMakeLists.txt

```diff
@@ -38,7 +40,7 @@ if(PYTHON_BINDINGS)
     endif()
     find_package(PythonInterp ${PYTHON_VERSION} REQUIRED)
     add_custom_target(test-python
-        COMMAND ${PYTHONPATH_CMD} ${PYTHON_EXECUTABLE} -m unittest discover ${CMAKE_SOURCE_DIR}/src/testers/unittests
+        COMMAND ${PYTHONPATH_CMD} ${PYTHON_EXECUTABLE} -m unittest discover ${CMAKE_CURRENT_LIST_DIR}/src/testers/unittests
```

Comment:
Why not using the new variable `TRITON_ROOT`?

Reply:
I guess that is mostly a stylistic question since they both refer to the same thing in this case but I can change them if you prefer